### PR TITLE
Owain/loss

### PR DESCRIPTION
**Pull request by owain-Biddulph**
Loss is RMSE, can be changed depending on what outputs we end up using

### CODE REVIEW

File: models/basic_cnn.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self, name="basic_cnn", subspectrogram_points = config.SUBSPECTROGR
 
         self.dense1 = Dense(200,activation=tf.nn.relu,name=f"{name}_dense1")
 
-        self.dense2 = Dense(number_of_emotions,activation=tf.nn.sigmoid,name=f"{name}_output")
```
Comment (quentinverlhac) [inline models/basic_cnn.py]: Are we sure about removing this ?

File: utils.py [hunk 2]
```diff
@@ -71,3 +71,7 @@ def setup_checkpoints(model, optimizer):
         max_to_keep=1
     )
     return checkpoint, checkpoint_manager
+
+
+def normalise_by_max(spectrogram):
+    return spectrogram / np.max(np.abs(spectrogram))
```
Comment (quentinverlhac) [inline utils.py]: Is it ok to normalize spectrograms ? Shouldn't we normalize audio files instead ?

File: models/basic_cnn.py (hunk 1 above)
Comment (Antoine-marchais) [inline models/basic_cnn.py]: You might want to keep the sigmoïd activation if the loss is mse

Comment (owain-Biddulph) [reply models/basic_cnn.py]: It is replaced not deleted, the activation has been put back in in another commit

Comment (owain-Biddulph) [reply models/basic_cnn.py]: Added back on

File: utils.py (hunk 2 above)
Comment (owain-Biddulph) [reply utils.py]: Why would it not be ok ?

Comment (quentinverlhac) [reply utils.py]: I've realized that normalizing FFT is probably the same as normalizing song amplitudes anyway. So let's keep it like that.
